templates/openshift/hello/dc/hello.yml.j2

```diff
@@ -6,7 +6,7 @@ metadata:
   name: hello
   namespace: "{{ project_name }}"
 spec:
-  replicas: 1  # number of pods we want
+  replicas: 2  # number of pods we want
```

Comment:
why 2 replicas ? Is for testing de blue/green switch ? the load-balancer ?

Reply:
yes load balancer and why not.